src/app/pages/dashboard/dashboard.component.ts

```diff
@@ -120,13 +126,16 @@ export class DashboardComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   ngOnInit(): void {
-    this.timerUnSub = timer(0, 60000)
-      .pipe(takeUntil(this.destroy$))
-      .subscribe(() => this.getInfoData());
-
+    this.getInfoData();
     this.initChart();
     this.getCoinInfo(this.chartRange);
     this.getVotingPeriod();
+
+    this.environmentService.nextHeight$.pipe(takeUntil(this.destroy$)).subscribe((res) => {
+      if (res) {
+        this.getInfoData();
```

Comment:
Truyền thêm height vào
```suggestion
        this.getInfoData(height);
```